wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+Cela permettrait aux joueurs novices et expérimentés de pouvoir jouer et apprécier la beatmap.
+
+Quelques-uns des facteurs qui peuvent affecter la difficulté d'une version sont le nombre d'objets dans la beatmap, qu'importe que le rythme choisi pour le placement des objets soie le rythme principal ou un rythme d'arrière plan plus difficile à entendre,la longueur des combos, la valeur d'espacement des pulsations, le taux de drainage, la limite de précision requise par le jeu pour enregistrer une frappe ,la taille du cercle de frappe, etc..
+Les étoiles de difficulté sont un moyen de jauger la version de difficulté d'un niveau.
+Le "osu!website" trie les difficultés en 5 catégories (en anglais):
+
+- Easy,
+- Normal,
+- Hard,
+- Insane, and
+- Expert
+
+### Temps de drainage (terme original : drain time) <!--Review needed | max priority -->
+
+Le temps de drainage, ou drain, se réfère à la durée totale de la beatmap qui requiert que le joueur soie actif.
+Le temps de drainage ne prend pas en compte les pauses et les parties non mappées à travers la beatmap.
+
+Les maps avec des temps de drainage très longs sont typiquement référencées comme étant des maps "marathon" et requièrent une endurance supérieure à la normale pour être complétées avec succès.
+
+### Drop  <!--Review needed | max priority -->
+
+Un drop est un élément d' "osu!catch" qui donne un score de 100.
+Un drop est l'équivalent du point d'entrée/sortie d'un slider sur osu!standard.
+
+La différence entre les drops et les droplets est qu'un drop est deux fois plus grand qu'un droplet.
+
+### Droplet
+
+Un droplet est un élément d' "osu!catch" qui donne un score de 10.
+Un droplet est l'équivalent du "rail" d'un slider sur osu!standard.
+
+La différence entre les drops et les droplets est qu'un droplet fait la moitié de la taille d'un drop.
+
+### DS
+
+DS est une abbréviation signifiant "distance snapping".
+Ce terme est généralement aperçu durant la phase de modding.
+
+### DS games
+
+Un terme désignant les jeux : [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan) (aussi appelé "Ouendan" or "Ouendan 1"), [Moero! Nekketsu Rhythm Damashii Osu! Tatakae! Ouendan! 2](https://en.wikipedia.org/wiki/Moero!_Nekketsu_Rhythm_Damashii_Osu!_Tatakae!_Ouendan!_2) (généralement appelé "Ouendan 2") and [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents) (aussi appelé "EBA").
+Il s'agit d'une collection de jeux de rythme développés par [iNiS](https://en.wikipedia.org/wiki/INiS) pour la Nintendo DS, qui se partagent beaucoup d'éléments de gameplay et de présentation.
+osu! est en grande partie basé sur ces jeux.
+
+## E
+
+### EBA
+
+EBA a deux sens:
+
+- Elite Beat Agents, un des jeux DS sur lesquels osu! est basé
+- un terme collectif désignant les personnages jouables dans le jeu ci-dessus, par exemple : "The EBA" ou "The Agents"
+
+### Chat étandu (terme original : extanded chat) -> https://github.com/ppy/osu-wiki/pull/100
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Chat étandu

_Terme original : extanded chat_
```